frontend/src/console/components/NoticeBoard/NoticeBoard.tsx

```diff
@@ -0,0 +1,127 @@
+
+import './styles.scss';
+
+interface ActionProps {
+  getAlerts: Function;
+}
+interface Props extends ActionProps {
+  alerts: Array<DeploymentAlertModel>;
+}
+
+interface AlertProps {
+  alert: DeploymentAlertModel;
+}
+
+const iconClassMap = {
+  [DeploymentStatus.Healthy]: 'fa-check-circle',
+  [DeploymentStatus.ChangesPending]: 'fa-check-circle',
+  [DeploymentStatus.Provisioning]: 'fa-info-circle',
+  [DeploymentStatus.Preparing]: 'fa-info-circle',
+  [DeploymentStatus.Unhealthy]: 'fa-times-circle',
+  [DeploymentStatus.Offline]: 'fa-stop-circle'
+};
+
+const statusClassMap = {
+  [DeploymentStatus.Healthy]: 'healthy',
+  [DeploymentStatus.ChangesPending]: 'changes_pending',
+  [DeploymentStatus.Provisioning]: 'provisioning',
+  [DeploymentStatus.Preparing]: 'preparing',
+  [DeploymentStatus.Unhealthy]: 'unhealthy',
+  [DeploymentStatus.Offline]: 'offline'
+};
+
+const Alert: React.FC<AlertProps> = (props: AlertProps) => {
+  const [detailsCollapsed, setDetailsCollapsed] = React.useState(true);
+
+  const deploymentStatus = props.alert.status;
+
+  const caretClass = detailsCollapsed ? 'fa-caret-down' : 'fa-caret-up';
+  const collapsedClass = detailsCollapsed ? 'collapsed' : '';
+  const iconClass = iconClassMap[deploymentStatus];
+  const statusClass = statusClassMap[deploymentStatus];
+
+  const month = props.alert.date.toLocaleString('default', { month: 'long' });
+  const day = props.alert.date.getDate();
+  const year = props.alert.date.getFullYear();
+
+  const details = props.alert.deployedChanges ? (
+    JSON.stringify(props.alert.deployedChanges)
+  ) : (
+    <WrappedMessage id="NoDetails" messages={messages} />
+  );
+
+  return (
+    <div className="deployment-alert mb-3">
+      <div className={`p-3 d-flex deployment-alert-header ${statusClass}`}>
+        <div className="d-flex">
+          <i className={`mr-3 align-self-center fa ${iconClass}`} />
+        </div>
+        <div>
+          <p className="m-0 deployment-alert-message">
+            <WrappedMessage id={deploymentStatus} messages={messages} />
+          </p>
+          <p className="m-0 deployment-alert-date">
+            <small>{`${month} ${day}, ${year}`}</small>
+          </p>
+        </div>
+        <div // eslint-disable-line jsx-a11y/interactive-supports-focus
+          role="button"
+          className="d-flex expand-caret-button ml-auto"
+          onClick={() => setDetailsCollapsed(!detailsCollapsed)}
+          onKeyDown={() => setDetailsCollapsed(!detailsCollapsed)}
+        >
+          <i className={`align-self-center fa ${caretClass}`} />
+        </div>
+      </div>
+      <div className={`deployment-alert-body ${statusClass} ${collapsedClass}`}>
+        <div className="m-3">{details}</div>
+      </div>
+    </div>
+  );
+};
+
+const NoticeBoardComponent: React.FC<Props> = (props: Props) => {
+  React.useEffect(
+    () => props.getAlerts(),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    []
+  );
```

Comment:
I think this is OK. I think it can be avoided by doing:
```jsx
React.useEffect(
    () => {props.getAlerts()},
    // eslint-disable-next-line react-hooks/exhaustive-deps
    []
  );
```